apps/ledger-live-desktop/src/renderer/families/cardano/UndelegateFlowModal/steps/StepSummary.tsx

```diff
@@ -96,18 +97,28 @@ export default class StepSummary extends PureComponent<StepProps> {
 }
 export function StepSummaryFooter({ transitionTo, status, bridgePending, transaction }: StepProps) {
   const { errors } = status;
-  const canNext = !bridgePending && !errors.validators && transaction;
+  const canNext = !errors.amount && !bridgePending && !errors.validators && transaction;
+  const displayError = errors.amount?.message ? errors.amount : "";
   return (
     <>
-      <Box horizontal>
-        <Button
-          id="undelegate-continue-button"
-          disabled={!canNext}
-          primary
-          onClick={() => transitionTo("connectDevice")}
-        >
-          <Trans i18nKey="common.continue" />
-        </Button>
+      <Box horizontal alignItems="center" flow={2} grow>
+        {displayError ? (
+          <Box grow>
```

Comment:
@KVNLS Is it an alert component that needs to be used here ? :thinking: I see `Alert type="error"` in the common component

Reply:
Yes please use `import Alert from "~/renderer/components/Alert";`

Reply:
@Prashanthsoordelu

Reply:
@KVNLS We want alert here on the screenshot ? I think it's supposed to be just under network fees ? (and that will be the case with generic UI ?)